masterlist.yaml

```diff
@@ -628,3 +628,12 @@ plugins:
 ###### 3RD Party Patches ######
 
 ###### Anything new can go after this (If you're not sure where to put it) ######
+
+###### Late Fixes & Changes ######
+  - name: 'final patch.esm'
+    url:
+      - link: 'https://www.nexusmods.com/starfield/mods/12072/'
+        name: 'The Final Patch (Nexus Mods)'
+      - link: 'https://creations.bethesda.net/en/starfield/details/04278b2d-3e4a-4b3c-8dee-0d98bab3ddad/The_Final_Patch/'
+        name: 'The Final Patch (Bethesda.net Mods)'
+    group: *lateChangesGroup
```

Comment:
```suggestion
  - name: 'final patch.esm'
    url:
      - link: 'https://www.nexusmods.com/starfield/mods/12072/'
        name: 'The Final Patch (Nexus Mods)'
      - link: 'https://creations.bethesda.net/en/starfield/details/04278b2d-3e4a-4b3c-8dee-0d98bab3ddad/The_Final_Patch/'
        name: 'The Final Patch (Bethesda.net Mods)'
    group: *lateChangesGroup

```

Reply:
I made the suggested changes. Thank you for your help.